examples/nft-nd/nft.yml

```diff
@@ -13,7 +13,19 @@ events:
         type: Integer
       - name: tokenId
         type: ByteArray
+  - name: RoyaltiesTransferred
+    parameters:
+      - name: royaltyToken
+        type: Hash160
+      - name: royaltyRecipient
+        type: Hash160
+      - name: buyer
+        type: Hash160
+      - name: tokenId
+        type: ByteArray
+      - name: amount
+        type: Integer
```

Comment:
Not for the NEP11 contract itself, read the [spec](https://github.com/neo-project/proposals/blob/master/nep-24.mediawiki#common-methods):
> Marketplaces that support this standard MUST emit the event, RoyaltiesTransferred for each recipient, after sending a payment.

Reply:
so I should remove it from manifest/standard too

Reply:
Yes, it's not a part of the standard that belongs to the NEP11 contract itself. But it should be moved to a separate standard, ref. https://github.com/nspcc-dev/neo-go/pull/3560#discussion_r1816447746.